src/v0/util/index.js

```diff
@@ -47,6 +47,12 @@ const flattenMap = (collection) => _.flatMap(collection, (x) => x);
 // GENERIC UTLITY
 // ========================================================================
 
+const getEndpointPath = (endpoint) => {
+  // extract the path details from the whole endpoint url passed
+  const url = new URL(endpoint);
+  return url.pathname.replace(/\$%7B(.*?)%7D/g, 'xx').replace(/^\/+/, '');
```

Comment:
what does this pattern match? we can replace with ""

Reply:
I matched with the pattern of adding dynamic variables, i.e `${variable_name}`, and replaced with xx.
Replacing it with "" as suggested

Reply:
Updated to destination specific handling from a common util fxn.